openslides/utils/migrations.py

```diff
@@ -20,7 +20,7 @@ def add_permission_to_groups_based_on_existing_permission(
 
     def function(apps: Any, schema_editor: Any) -> None:
         content_type = ContentType.objects.filter(model=model, app_label=app_label)
-        base_perm = Permission.objects.filter(codename=codename, content_type=content_type)
+        base_perm = Permission.objects.filter(codename=codename, content_type__in=content_type)
```

Comment:
Maybe line 22 must be "get" instead of "filter"?

Reply:
it has to be filter because it can be an empty list. Now it is the same as before.